server/bundler/extensions-loader.js

```diff
@@ -0,0 +1,34 @@
+const existsSync = require( 'fs' ).existsSync;
+const path = require( 'path' );
+const camelCase = require( 'lodash' ).camelCase;
+
+function getReducers( extensions = {} ) {
+	return extensions
+		.filter( ( extension ) => existsSync( path.join( 'client', 'extensions', extension, 'state', 'reducer.js' ) ) )
```

Comment:
this applies a native filter and map over a value which according to the function declaration is an object. it _should_ throw a `TypeError`

was `extensions` supposed to default to `[]` instead?